### ISSUE TITLE
[feature]: Modal shouldn't get closed when clicked outside while creating an issue

### ISSUE THREAD

**Issue by devmo01** [makeplane/plane#1809]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

When we are creating an issue modal shouldn't get closed if clicked outside (backdrop). It should get only closed on if user clicks on discard button

![image](https://github.com/makeplane/plane/assets/141747557/475d7979-05cc-4348-b881-d064c1e7042f)

![image](https://github.com/makeplane/plane/assets/141747557/b24092a2-4c63-4d1f-9462-60f5b70edeb0)

### Why should this be worked on?

When user is creating issue and writing length descriptions or in a mid of writing description, clicking outside  by mistake  everything is lost and irritates them. 

Closing modal by clicking outside makes lot's of sense but not in this scenario.

**Comment by erhan-** [comment]
It still makes sense but it should ask if you want to discard changes when you click outside if it does not yet.

**Comment by yiiman-dev** [comment]
> ### Is there an existing issue for this?
> * [x]  I have searched the existing issues
> 
> ### Summary
> When we are creating an issue modal shouldn't get closed if clicked outside (backdrop). It should get only closed on if user clicks on discard button
> 
> ![image](https://private-user-images.githubusercontent.com/141747557/259132181-475d7979-05cc-4348-b881-d064c1e7042f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ4OTY0NTMsIm5iZiI6MTcxNDg5NjE1MywicGF0aCI6Ii8xNDE3NDc1NTcvMjU5MTMyMTgxLTQ3NWQ3OTc5LTA1Y2MtNDM0OC1iODgxLWQwNjRjMWU3MDQyZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwNVQwODAyMzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYmZhMjRmMDNhYWU4OTQ5OWYzOTQ5MmQ4NzkyMGIwYzkwNDU4MDhiZjZhZTUxMDZjNTM3NTNlYzExZmNlY2QxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.lSN0yHqvXeCfOFwlfa8UNLpGG2YBPRl-yYhXIYQQOBA)
> 
> ![image](https://private-user-images.githubusercontent.com/141747557/259132363-b24092a2-4c63-4d1f-9462-60f5b70edeb0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ4OTY0NTMsIm5iZiI6MTcxNDg5NjE1MywicGF0aCI6Ii8xNDE3NDc1NTcvMjU5MTMyMzYzLWIyNDA5MmEyLTRjNjMtNGQxZi05NDYyLTYwZjViNzBlZGViMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwNVQwODAyMzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNGNlMzYxZWQxN2NlNWNiZDM3YzA1NGU5MDlkZTBmMjRlYTQ3MWQ4ZDJhYjRmYzkzNjlhMzcyNjg5ZjM4ZjhiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.KsxA05z0HwBke7D_6bX5H2AizQcxZD93lhZeNbYUv4Q)
> 
> ### Why should this be worked on?
> When user is creating issue and writing length descriptions or in a mid of writing description, clicking outside by mistake everything is lost and irritates them.
> 
> Closing modal by clicking outside makes lot's of sense but not in this scenario.

I have this problem, too
Thanks for report this issue

**Comment by UndefinedYara** [comment]
@aheckmann Good day! can this be assigned to me? I'm thinking of trading off between just issuing a warning popover when clicked outside the issue popup and just not closing it at all. I would love to get both versions working and then we could take it from there

**Comment by erhan-** [comment]
I think not closing at all might be good. If you want to close it, there is a button for it called Discard.

**Comment by monis01** [comment]
Yes, so the user is safe in continuing the work without getting overwhelmed by another pop up asking for another confirmation.

**Comment by vihar** [comment]
This has been the behavior for a while now. Clicking outside the create modal doesn't close it anymore, and hitting Discard with unsaved changes prompts you to save to Drafts or discard. Closing.
